doc/community/contributing.md

```diff
@@ -0,0 +1,111 @@
+# How to Contribute
+
+Contributions are welcome, and they are greatly appreciated!
+Every little bit helps, and you always earn credits.
+
+You can contribute on the code side in many ways:
+
+* submit feedback,
+* add new features,
+* report bugs,
+* fix bugs,
+* implement a new cluster/cloud computation backend,
+* write documentation
+
+## Code
+
+### Linting and pre-commit
+
+For every code contribution you should run [pre-commit](https://pre-commit.com/index.html).
+This will lint, format and check your code contributions against our guidelines
+(e.g. we use [Black](https://github.com/psf/black) as code style
+and aim for [REUSE compliance](https://reuse.software/)):
+
+1. Installation `conda install -c conda-forge pre-commit` or `pip install pre-commit`
+2. Usage:
+   * To automatically activate `pre-commit` on every `git commit`: Run `pre-commit install`
+   * To manually run it: `pre-commit run --all`
+
+## Documentation
+
+### How to docs?
+
+We add the code documentation along the way.
+It might seem time-consuming and inefficient, but that's not really true anymore!
+Documenting with great tools makes life much easier for YOU and YOUR COLLABORATORS and speeds up the overall process.
+Using [MkDocs](https://www.mkdocs.org/) and its plugin
+[mkdocstrings](https://mkdocstrings.github.io/) we document in our
+code scripts which then will automatically generate the documentation you might see here.
```

Comment:
I'd suggest to shift a bit a focus from magnificent properties of `MkdDocs` (which are certainly worth proper acknowledgement) to the need of the project. For example:
```suggestion
We add the code documentation in a dynamic way along with the growth of the project. 
The documenting tools we are using [MkDocs](https://www.mkdocs.org/) and [mkdocstrings](https://mkdocstrings.github.io/) make documenting easy. Feel free to check it yourself by
starting to contribute. Every typo counts!
```